docs/guide/module-hierarchy.rst

```diff
@@ -253,17 +286,28 @@ Spark <http://spark.apache.org/>`__.
    coordinates.
 -  Utilities around creating spark contexts for applications using
    GeoTrellis, including a Kryo registrator that registers most types.
+-  Implements GeoTrellis COGLayers creation, persistence and query mechanisms.
 
 geotrellis-spark-pipeline
 -------------------------
 
-TODO: Complete
+Pipelines are the operative construct in GeoTrellis,
+the original idea was taken from `PDAL <https://pdal.io/pipeline.html>`__.
+Pipelines can represent not a simple ETL process but a set of instructions:
```

Comment:
Suggest:

> Pipelines can represent not a simple ETL process but a set of instructions:

becomes

> Pipelines represent a set of instructions rather than a simple ETL process: